src/test/scala-2.11/markets/clearing/engines/CDAMatchingEngineSpec.scala

```diff
@@ -326,7 +326,7 @@ class CDAMatchingEngineSpec extends TestKit(ActorSystem("CDAMatchingEngineSpec")
 
       // also need to check that residual ask order landed in the book
       val residualAskQuantity = askQuantity - bidQuantity
-      val residualAskOrder = askOrder.split(residualAskQuantity)
+      val (_, residualAskOrder) = askOrder.split(residualAskQuantity)
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Field name does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1537948230/issues/source?bid=2921361&fileBranchId=3014278#l329)